.vault-config/product-construction-int.yaml

```diff
@@ -43,8 +43,3 @@ secrets:
       hasPrivateKey: true
       hasWebhookSecret: false
       hasOAuthSecret: true
-
-  maestro-token:
```

Comment:
Please also remove it from the KeyVaults

Reply:
removed